js/src/bindings.ts

```diff
@@ -127,6 +124,7 @@ export const placeOrder = async (
   selfTradeBehaviour: SelfTradeBehavior,
   ownerTokenAccount: PublicKey,
   owner: PublicKey,
+  clientOrderId: BN,
```

Comment:
Should prob be optional i.e `clientOrderId?: BN,` and if `undefined` be `crypto.randomBytes(16)`